src/var/isAttached.js

```diff
@@ -0,0 +1,8 @@
+define( function() {
+	"use strict";
+
+	return function isAttached( obj ) {
+		return jQuery.contains( obj.ownerDocument, obj );
+  };
```

Comment:
I'm surprised this passed lint. Looks like a tabs/spaces mixture here.

Reply:
Really sorry for this. I am not sure how this mixture happened.

Reply:
No worries. I'm just concerned that travis didn't pick it up.